README.md

```diff
@@ -63,6 +74,11 @@ If you would like to use the helpers without a prefix, so just by name, use this
 HandlebarsHelpers.Register(handlebarsContext, options => { UseCategoryPrefix = false; });
 ```
 
+Now you can use it like:
+``` handlebars
+{{[String-Append] "foobar" "bar"}}

```

Comment:
nit: You do not need to wrap helper ref with [ ] in case valid separator is used.

Reply:
Correct. FIxed.